contracts/TopShot.cdc

```diff
@@ -1286,8 +1319,8 @@ pub contract TopShot: NonFungibleToken {
         // Parameters: id: The ID of the NFT to get the reference for
         //
         // Returns: An optional reference to the desired NFT, will be nil if the passed ID does not exist
-        pub fun borrowNFTSafe(id: UInt64): &NonFungibleToken.NFT? {
-            if let nftRef = &self.ownedNFTs[id] as &NonFungibleToken.NFT? {
+        view access(all) fun borrowNFTSafe(id: UInt64): &{NonFungibleToken.NFT}? {
```

Comment:
`borrowNFTSafe` can be completely removed since `borrowNFT` does the same thing now